vernac/comAssumption.mli

```diff
@@ -61,7 +61,7 @@ val declare_global
   -> univs:UState.named_universes_entry
   -> impargs:Impargs.manual_implicits
   -> inline:Declaremods.inline
-  -> name:variable
+  -> lident
```

Comment:
Why the change from labelled to non-labelled?

Reply:
Labelled was more syntactically heavy and IMO is not beneficial here since we have the type (unlike eg a labelled `bool` argument)

Reply:
ok changed